operator/src/main/java/com/instaclustr/cassandra/operator/service/ControllerService.java

```diff
@@ -164,50 +164,47 @@ private void createOrReplaceStateNodesStatefulSet(final DataCenter dataCenter, f
                                         .addContainersItem(new V1Container()
                                                 .name(dataCenterMetadata.getName())
                                                 .image(dataCenter.getSpec().getImage())
-                                                .imagePullPolicy("Never")
+                                                .imagePullPolicy(dataCenter.getSpec().getImagePullPolicy())
```

Comment:
I think you're assuming the break point is - someone defines param `imagePullPolicy` but gives a invalid string (like empty or values not applicable)?

Reply:
Thinking about it more, we probably don't need to do any validation on our end since it's just pass-through. If the user specifies an invalid pull policy or overrides the schema validation and provides null, then k8s itself will complain, which will then result in our operator throwing an error too.